### PR TITLE
[exporter/dynatrace] improve temporality documentation

### PR DESCRIPTION
**Pull request by dyladan**
Improve the documentation on supported temporality and delta temporality preference for Dynatrace exporter.

### CODE REVIEW

**Comment by dyladan** [comment]
I think this change doesn't require a changelog update.

**Comment by dyladan** [comment]
Integration test failure shouldn't be related since this is just markdown

**Review by arminru — APPROVED** [review]

File: exporter/dynatraceexporter/README.md [hunk 1]
```diff
@@ -265,17 +265,31 @@ Default: `5000`
...
-or inconsistent data, as described below:
+If possible when configuring your SDK, use DELTA temporality for Counter, Asynchronous Counter, and Histogram metrics.
+Use CUMULATIVE temporality for UpDownCounter and Asynchronous UpDownCounter metrics.
+When using OpenTelemetry SDKs to gather metrics data, setting the
+`OTEL_EXPORTER_OTLP_METRICS_TEMPORALITY_PREFERENCE` environment variable to `delta`
+should correctly set temporality for all metrics.
+You can check the [spec compliance matrix](https://github.com/open-telemetry/opentelemetry-specification/blob/main/spec-compliance-matrix.md#environment-variables)
+if you are unsure if the SDK you are using supports this configuration.
+You can read more about this and other configurations at
+https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk_exporters/otlp.md#additional-configuration.
```
Comment (joaopgrassi) [inline exporter/dynatraceexporter/README.md:278]: Maybe a "formatted link" makes it easier to read? 
```suggestion
[OpenTelemetry Metrics Exporter - OTLP](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk_exporters/otlp.md#additional-configuration).
```

**Review by joaopgrassi — APPROVED** [review]

**Comment by dyladan** [comment]
Can someone with permission add the skip changelog label?

**Comment by dyladan** [comment]
ping @open-telemetry/collector-contrib-maintainer I believe this can have skip changelog label applied and be merged?

**Review by mx-psi — APPROVED** [review]

**Pull request merged**